bindings/python/examples/dnn/dnn.py

```diff
@@ -115,8 +121,8 @@ class ModesEnum(Enum):
     smallSignalThreshold = 100
     cameras[0].setControl("noise_reduction_threshold", str(smallSignalThreshold))
 
-    camera_range = camDetails.maxDepth
```

Comment:
The fields maxDepth and bitCount are not returning the correct depth and bit count values. Needed to be further investigated.

Reply:
maxDepth and bit count are not populated by the SDK.